.gitignore

```diff
@@ -8,6 +8,7 @@ src/strategies/__pycache__/
 *.pyc
 .env
 update.sh
+.python-version
```

Comment:
Why is it worth removing `.python-version`? 🙂

Reply:
`.python-version` is a file used by pyenv and it’s standard practice to add it to the `.gitignore` so I did that here. In this case I needed the file because my default Python is 3.9 so I needed to tell pyenv to use 3.8. The two main reasons it is typically added to `.gitignore` are only pyenv knows how to use it so it won’t do anything if you aren’t using pyenv (this one isn’t a big deal). The bigger reason is it is very specific to the Python versions installed. For example I have Python 3.8.8 in the `.python-version`, but maybe you have 3.8.7 installed on your machine. Both Python versions work but yours would not be able to run with my `.python-version` file and you would get an error. 

This brings up another question I was going to ask. Is there a reason only 3.8 is supported? If I was to test 3.9 without issues would you be interested in adding that since it is the most recent version of Python?

Reply:
Ah sorry, I might be tired, I thought this was a deletion!! 😂 
It makes sense!

We don't support python 3.9 because we didn't add the support yet (add it to the CI) but I will open an issue! :)